src/components/Badge/style.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../scss/variables/colors';
+
+.badge {
+  position: relative;
+  display: inline-block;
+  vertical-align: middle;
+  height: 16px;
```

Comment:
imho instead of height u should use `padding` to control size of the component. Also, padding left and right doesn't great when content is longer http://take.ms/qifur